### DISCUSSION TITLE
Greater London archaeological priority areas

### DISCUSSION THREAD

**Discussion by kieranwint** [digital-land/data-standards-backlog#89] [Planning consideration]
Areas of Greater London where there is significant known archaeological interest or potential for new discoveries.

We are looking at this consideration as there is potential for there to be national data available. We are looking to do the research to see whether any potential dataset would be usable.

**Comment by colmjude** [comment]
## Analysing the existing data

[Historic England publish this data](https://opendata-historicengland.hub.arcgis.com/datasets/historicengland::greater-london-archaeological-priority-areas-apas/about) under OGL.

We have analysed the dataset.

### Shapefile summary

* **Geometry Type**: Polygon
* **Number of Records**: 1015
* **Features/Attributes**: area_type, descriptio, designatio, primary_re, resourcein, geometry
* **Coordinate Reference System (CRS)**: PROJCS["OSGB36_Highways_England_B16H4 (deprecated)",GEOGCS["OSGB36",DATUM["Ordnance_Survey_of_Great_Britain_1936",SPHEROID["Airy 1830",6377563.396,299.3249646,AUTHORITY["EPSG","7001"]],AUTHORITY["EPSG","6277"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4277"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",49],PARAMETER["central_meridian",-2],PARAMETER["scale_factor",1.000087],PARAMETER["false_easting",54026.22785],PARAMETER["false_northing",111053.9128],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH],AUTHORITY["ESRI","102862"]]
* **File Size**: 2149056 bytes

It is worth noting the use of a deprecated CRS.
Deprecated projected coordinate system = OSGB36_Highways_England_B16H4

### Example data / record

GeoJSON
```
{
"type": "FeatureCollection",
"name": "sample",
"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:ESRI::102862" } },
"features": [
{ "type": "Feature", "properties": { "area_type": null, "descriptio": "The Saxon place name is apparently derived from â€˜Beddaâ€™s settlementâ€™. The Domesday Book refers to two mills and a church, probably the original foundation of St. Maryâ€™s. Two stone sarcophagi found in the churchyard of St. Maryâ€™s reveal the pres", "designatio": "Archaeological Priority Area", "primary_re": "76962", "resourcein": "60d85e7b-9e89-43db-9af6-2ee585381d69" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 184106.911640385078499, 376081.417893437552266 ], [ 184076.469677281391341, 376073.476620264933445 ], [ 184042.056978648266522, 376073.476687863701954 ], [ 184012.938411826791707, 376074.800846500496846 ], [ 183977.202305176935624, 376077.447809292352758 ], [ 183921.612976747681387, 376066.859325987228658 ], [ 183821.021660392987542, 376045.682954568590503 ], [ 183731.019634970813058, 376031.123873425531201 ], [ 183691.312956371693872, 376024.506076425779611 ], [ 183656.900290110323112, 376024.506154376547784 ], [ 183654.252957570337458, 376074.802824481215794 ], [ 183592.045558224257547, 376073.479229068208952 ], [ 183589.398234170745127, 376058.919684839318506 ], [ 183570.868270643084543, 376058.919794024259318 ], [ 183533.808871703688055, 376092.009735268016811 ], [ 183502.043493019271409, 376089.362635223136749 ], [ 183457.042238270398229, 376089.362908377195708 ], [ 183451.747584858414484, 376068.185818013560493 ], [ 183455.718231697159354, 376052.302872877917252 ], [ 183329.980179188452894, 376021.860842523921747 ], [ 183271.743548805214232, 375999.359939881949686 ], [ 183241.301546146918554, 375984.800545684935059 ], [ 183217.476915688690497, 375971.565040427551139 ], [ 183185.711577821581159, 375964.946885143173859 ], [ 183135.416228412766941, 376125.100776677543763 ], [ 183135.416238453355618, 376135.68866268260171
```

Attributes

| field | value |
| --- | --- |
| area_type | null |
| descriptio | The Saxon place name is apparently derived from â€˜Beddaâ€™s settlementâ€™. The Domesday Book refers to two mills and a church, probably the original foundation of St. Maryâ€™s. Two stone sarcophagi found in the churchyard of St. Maryâ€™s reveal the pres |
| designatio | Archaeological Priority Area |
| primary_re | 76962 |
| resourcein | 60d85e7b-9e89-43db-9af6-2ee585381d69 |

### Example plots

Single archaeological priority area
![glapa-single-plot](https://github.com/user-attachments/assets/5c5a908e-09ae-47f0-bd27-9f6c49c1426e)

All archaeological priority areas
![glapa-london-plot](https://github.com/user-attachments/assets/d4e5c2da-2a7e-4a4e-8818-f49fe4e08a70)

Archaeological priority areas by "tier"
![glapa-plot-by-tier](https://github.com/user-attachments/assets/ab621c63-4a29-4cd7-aa51-3b3c7b69c551)

**Reply by colmjude** [reply]
### Mappings

We have created a [archaeological-priority-area](https://github.com/digital-land/specification/blob/main/content/dataset/archaeological-priority-area.md?plain=1) schema.

Mapping should be:

| field | mapped from | default value |
| --- | --- | --- |
| `reference` | primary_re | |
| `name` | | |
| `description` | descriptio | |
| `archaeological-risk-tier` | designatio | |
| `geometry` | geometry | |
| `organisation` | | government-organisation:PB1164 |
| `entry-date` | | date first collected |
| `start-date` | | |
| `end-date` | | |

**Reply by colmjude** [reply]
As we learn more about the archaeological-risk-tier categories we will likely create a category dataset for these.

**Reply by colmjude** [reply]
This [data is now on the platform](https://www.planning.data.gov.uk/dataset/archaeological-priority-area).

Where there were records with identical `reference`, `name` and `description` we combined them into a single entity. Therefore, we have 738 entities from the 1015 records found.
